crates/transaction-pool/src/config.rs

```diff
@@ -57,3 +63,36 @@ impl Default for SubPoolLimit {
         }
     }
 }
+
+/// Price bump config (in %) for the transaction pool underpriced check.
+#[derive(Debug, Clone)]
+pub struct PriceBumpConfig {
+    /// Default price bump (in %) for the transaction pool underpriced check.
+    pub default_price_bump: PriceBump,
+    /// Replace blob price bump (in %) for the transaction pool underpriced check.
+    pub replace_blob_tx_price_bump: PriceBump,
+}
+
+impl Default for PriceBumpConfig {
+    fn default() -> Self {
+        Self {
+            default_price_bump: Default::default(),
+            replace_blob_tx_price_bump: PriceBump(REPLACE_BLOB_PRICE_BUMP),
+        }
+    }
+}
+
+#[derive(Debug, Clone)]
+pub struct PriceBump(u128);
```

Comment:
while I like having `default::default`,

I think this adds more complexity than it is useful,

I'm okay with just using u128 as fields ins PriceBumpConfig